common/search/src/test/scala/weco/api/search/elasticsearch/ElasticsearchServiceTest.scala

```diff
@@ -195,52 +275,100 @@ class ElasticsearchServiceTest
         val findByMultiSearchFuture = elasticsearchService
           .findByMultiSearch[ExampleThing](multiSearchRequest)
 
-        whenReady(findByMultiSearchFuture) {
-          case (errors, foundBySearch) =>
-            errors should have length (0)
-            foundBySearch.toSet shouldBe thingsToQueryFor.toSet
+        whenReady(findByMultiSearchFuture) { results =>
+          val (errorEither, foundBySearchEither) = results.partition(_.isLeft)
+          val errors = errorEither.map(_.left.value)
+          val foundBySearch = foundBySearchEither.map(_.right.value)
```

Comment:
Could we use a collect here?

```suggestion
          val errors = results.collect { case Left(err) => err }
          val foundBySearch = result.collect { case Right(found) => found }
```

Reply:
Running over these with with a map extracts their value but also verifies their handedness (?) using `EitherValues`.